geometry_msgs/PoseStamped pose_stamped
---
geometry_msgs/PoseStamped[] pose_stamped
